docs/tutorial.md

```diff
@@ -0,0 +1,557 @@
+
+      application {
+        Window(title = "Inbox", onCloseRequest = ::exitApplication) {
+          CircuitCompositionLocals(circuit) {
+            // ...
+          }
+        }
+      }
+    }
+    ```
+
+=== "JS"
+    ```kotlin title="main.kt"
+    fun main() {
+      val circuit = Circuit.Builder()
+        // ...
+        .build()
+
+      onWasmReady {
+        Window("Inbox") {
+          CircuitCompositionLocals(circuit) {
+            // ...
+          }
+        }
+      }
+    }
+    ```
+
+## `CircuitContent`
+
+`CircuitContent` ([docs](https://slackhq.github.io/circuit/api/0.x/circuit-foundation/com.slack.circuit.foundation/-circuit-content.html)) is a simple composable that takes a `Screen` and renders it.
+
+```kotlin title="CircuitContent"
+CircuitCompositionLocals(circuit) {
+  CircuitContent(InboxScreen)
+}
+```
+
+Under the hood, this instantiates the corresponding `Presenter` and `Ui` from the local `Circuit` instance and connects them together. All you need to do is pass in the `Screen` you want to render!
+
+This is the most basic way to render a `Screen`. These can be top-level UIs or nested within other UIs. You can even have multiple `CircuitContent` instances in the same composition.
+
+## Adding navigation to our app
+
+An architecture isn't complete without navigation. Circuit provides a simple navigation API that's focused around a simple `BackStack` ([docs](https://slackhq.github.io/circuit/api/0.x/backstack/com.slack.circuit.backstack/-back-stack/index.html)) that is navigated via a `Navigator` interface ([docs]()). In most cases, you can use the built-in `SaveableBackStack` implementation ([docs](https://slackhq.github.io/circuit/api/0.x/backstack/com.slack.circuit.backstack/-saveable-back-stack/index.html)), which is saved and restored in accordance with whatever the platform's `rememberSaveable` implementation is.
+
+```kotlin title="Creating a backstack and navigator"
+val backStack = rememberSaveableBackStack { 
+  // Push your root screen
+  push(InboxScreen)
+}
+val navigator = rememberCircuitNavigator(backStack) {
+  // Do something when the root screen is popped, usually exiting the app
+}
+```
+
+Once you have these two components created, you can pass them to an advanced version of `CircuitContent` that supports navigation called `NavigableCircuitContent` ([docs](https://slackhq.github.io/circuit/api/0.x/circuit-foundation/com.slack.circuit.foundation/-navigable-circuit-content.html)).
+
+```kotlin title="NavigableCircuitContent"
+NavigableCircuitContent(navigator = navigator, backstack = backStack)
+```
+
+This composable will automatically manage the backstack and navigation for you, essentially rendering the "top" of the back stack as your _navigator_ navigates it. This also handles transitions between screens ([`NavDecoration`](https://slackhq.github.io/circuit/api/0.x/backstack/com.slack.circuit.backstack/-nav-decoration/index.html)) and fallback behavior with unavailable routes.
```

Comment:
Might be a bit easier to find the fallback handling using the `unavailableRoute` parameter name.
```suggestion
This composable will automatically manage the backstack and navigation for you, essentially rendering the "top" of the back stack as your _navigator_ navigates it. This also handles transitions between screens ([`NavDecoration`](https://slackhq.github.io/circuit/api/0.x/backstack/com.slack.circuit.backstack/-nav-decoration/index.html)) and fallback behavior with `unavailableRoute`.
```

Reply:
Good point. Let me actually make it a link to the Circuit builder prop